rbd/admin/groupsschedule.go

```diff
@@ -0,0 +1,42 @@
+//go:build !nautilus
+// +build !nautilus
+
+package admin
+
+import (
+	ccom "github.com/ceph/go-ceph/common/commands"
+	"github.com/ceph/go-ceph/internal/commands"
+)
+
+// GroupSnapshotScheduleAdmin encapsulates management functions for
+// ceph rbd mirror group snapshot schedules.
+type GroupSnapshotScheduleAdmin struct {
+	conn ccom.MgrCommander
+}
+
+// GroupSnapshotSchedule returns a GroupSnapshotScheduleAdmin type for
+// managing ceph rbd mirror group snapshot schedules.
+func (ra *RBDAdmin) GroupSnapshotSchedule() *GroupSnapshotScheduleAdmin {
+	return &GroupSnapshotScheduleAdmin{conn: ra.conn}
+}
+
+// Add a new group snapshot schedule to the given pool/group based on the supplied
+// level spec.
+//
+// Similar To:
+//
+//	rbd mirror group snapshot schedule add <level_spec> <interval> <start_time>
```

Comment:
```suggestion
// Similar To:
//
//  rbd mirror group snapshot schedule add <level_spec> <interval> <start_time>
```
it looks like we have extra spaces here

Reply:
I am not sure, I don't see any change by either using a `tab` or a `space` to adjust it :/
It's same for mirror image schedule [here](https://github.com/red-hat-storage/go-ceph/blob/master/rbd/admin/msschedule.go#L44)

Reply:
![image](https://github.com/user-attachments/assets/14b6e4fd-b0ed-4bc5-8d2a-6f83f65d12f5)
it rendered as something like this in my vscode, we can leave it as it is

Reply:
Yeah, same for me.